python_modules/dagster/dagster/_core/definitions/asset_spec.py

```diff
@@ -293,81 +293,95 @@ def with_io_manager_key(self, io_manager_key: str) -> "AssetSpec":
             metadata={**self.metadata, SYSTEM_METADATA_KEY_IO_MANAGER_KEY: io_manager_key}
         )
 
+    @public
+    def replace_attributes(
+        self,
+        *,
+        key: CoercibleToAssetKey = ...,
+        deps: Optional[Iterable["CoercibleToAssetDep"]] = ...,
+        description: Optional[str] = ...,
+        metadata: Optional[Mapping[str, Any]] = ...,
+        skippable: bool = ...,
+        group_name: Optional[str] = ...,
+        code_version: Optional[str] = ...,
+        freshness_policy: Optional[FreshnessPolicy] = ...,
```

Comment:
can we remove freshness_policy and automation_condition to avoid needing to go through a deprecation cycle on them?